update/genesis/import.go

```diff
@@ -359,6 +398,7 @@ func (si *stateImport) saveRootHash(accountsDB state.AccountsAdapter, accType Ty
 		accountsDB.SnapshotState(rootHash)
 	}
 
+	log.Info("imported state rootHash", "rootHash", rootHash, "shID", shardID)
```

Comment:
"imported state" is enough, "rootHash" is printed after

Reply:
done